### PR TITLE
Fix contributor screen flash

### LINKED ISSUE #223: [Assigned] Contributor screen flashes on redisplay
## Overview (Required)
- Title says everything
- Probably Groupie Item equals() returns false or not call super constructor with id.

### PR DESCRIPTION
**Pull request by k-kagurazaka**
## Issue
- close #223 

## Overview (Required)
- Specify the id (contributor's name hash) to Groupie `Item` constructor to support diff calculation.
- Use `Section#add` instead of `Section#setHeader` like `SessionHeaderItem` and `DateHeaderItem` since Groupie's diff calculation may fail.

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/item/ContributorHeaderItem.kt [hunk 1]
```diff
@@ -22,4 +23,12 @@ class ContributorHeaderItem(
     override fun bind(viewBinding: ItemContributorHeaderBinding, position: Int) {
         viewBinding.contributorCount = contributorCount
     }
+
+    override fun isSameAs(other: Item<*>?): Boolean =
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/item/ContributorHeaderItem.kt]: 📝

Comment (takahirom) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/item/ContributorHeaderItem.kt]: 📝

**Comment by konifar** [comment]
Thanks! I learned groupie a lot from your PR! 😄

**Comment by takahirom** [comment]
Looks good! I don't know `isSameAs()`! 👍 Thanks!

**Pull request merged**